core-bundle/contao/library/Contao/InsertTags.php

```diff
@@ -181,7 +181,7 @@ static function ($allowedTag)
 			// Skip certain elements if the output will be cached
 			if ($blnCache)
 			{
-				if ($elements[0] == 'date' || $elements[0] == 'form_session_data' || $elements[0] == 'fragment' || ($elements[1] ?? null) == 'referer' || strncmp($elements[0], 'cache_', 6) === 0)
+				if ($elements[0] == 'date' || $elements[0] == 'form_session_data' || $elements[0] === 'form_confirmation_message' || $elements[0] == 'fragment' || ($elements[1] ?? null) == 'referer' || strncmp($elements[0], 'cache_', 6) === 0)
```

Comment:
@ausi is this okay until https://github.com/contao/contao/pull/4779 is ready?

Reply:
I think so, yes.